reviews_ratings.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask, request, jsonify
+
+app = Flask(__name__)
+
+# Global variables
+reviews = []
```

Comment:
**Scalability Issue**: Using a global list 'reviews' to store review data is not scalable. As the number of reviews grows, this will consume more memory and could lead to performance issues. <br> **Fix**: Use a database system to store review data instead of an in-memory list. This will ensure that the data is managed more efficiently and can scale with the number of users and reviews. <br> **Code Suggestion**: 
 ```
 -reviews = []
 +from sqlalchemy import create_engine
 +from sqlalchemy.orm import sessionmaker
 +from your_model import Review  # Assuming you have a model class 'Review'
 +
 +# Setup database connection
 +engine = create_engine('sqlite:///reviews.db')  # Example using SQLite
 +Session = sessionmaker(bind=engine)
 +session = Session()
 ```